src/bin/es/ingest.mjs

```diff
@@ -1,11 +1,13 @@
+import * as _ from 'lamb';
 import { promises as fs } from 'fs';
 
 import { Command } from 'commander';
-import * as _ from 'lamb';
-
+import { bulkRequest } from 'es/bulk.mjs'
 import { arxliveCopy } from 'conf/config.mjs';
 import { createIndex } from 'es/index.mjs'
-import { bulkRequest } from 'es/bulk.mjs'
+import { logger } from 'logging/logging.mjs'
+import { batch } from 'util/array.mjs'
+import { commanderParseInt } from 'util/commander.mjs'
```

Comment:
2) imports should be formatted as usual  (node, npm, local, each block separated by a single line and sorted by module name/path)

```js
import { promises as fs } from 'fs';

import { Command } from 'commander';
import * as _ from 'lamb';

import { arxliveCopy } from 'conf/config.mjs';
import { bulkRequest } from 'es/bulk.mjs';
import { createIndex } from 'es/index.mjs';
import { logger } from 'logging/logging.mjs';
import { batch } from 'util/array.mjs';
import { commanderParseInt } from 'util/commander.mjs';
```